### PR TITLE
Plate projections

### PR DESCRIPTION
**Pull request by MattsonCam**
In this pr, plates 1 and 2 data are projected to visualize multiple relationships. Please let me know if you have any questions or suggestions!

### CODE REVIEW

File: 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+""" This file provides analysis utilities for a variety of tasks """
```
Comment (gwaybio) [inline 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py]: why is this file saved in `nbconverted` folder? Is it originally a `.ipynb` file?

I typically expect to only see nbconverted files in here, not utility scripts (they typically go in a `utils` or `scripts` folder

File: 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import numpy as np
+import pandas as pd
+from scipy.stats import f_oneway
+import itertools
+
+rnd_val = 0  # Random value for all seeds
+rng = np.random.default_rng(seed=rnd_val)  # random number generator
+
+
+def plot_pca(
```
Comment (gwaybio) [inline 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py]: I thought I saw these already in #7 ? Do you know what is going on? Maybe the same commits on duplicate PRs?

Comment (gwaybio) [reply 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py]: in general I am a big fan of these importable functions!

**Review by gwaybio — APPROVED** [review]
It looks like #7 included the first two commits here

Screen Shot 2023-06-28 at 10 29 54 AM

I provided some minor additional commentary here, but I reviewed it in #7 

We can chat tomorrow about github usage patterns when opening multiple PRs in the same project, if you'd like :)

**Review by MikeLippincott — COMMENTED** [review]
It looks like this was in PR 7, as Greg mentioned.

File: 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py (hunk 1 above)
Comment (MattsonCam) [reply 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py]: Good catch, forgot to add the file to a utils directory

File: 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py (hunk 2 above)
Comment (MattsonCam) [reply 0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py]: You did, yes, one branch comes from the other. 

They can be useful, like in these cases!

**Pull request merged**